flow-http/build.gradle

```diff
@@ -12,7 +12,7 @@ buildscript {
         log4j_version = constants.getProperty("log4jVersion")
         slf4j_version = constants.getProperty("slf4jVersion")
         corda_platform_version = constants.getProperty("platformVersion")
-        okhttp_version = '3.5.0'
+        okhttp_version = '3.14.1'
```

Comment:
I'm a bit confused. What prompted this change is the update in corda master, which we presume is happening because it's pulling this in as a transitive dependency. If that's the case then this `okhttp_version` is not used and can be deleted?